core/src/main/java/org/apache/struts/util/ResponseUtils.java

```diff
@@ -82,8 +87,11 @@ public static String filter(String value) {
                 break;
 
             case '&':
-                filtered = "&amp;";
-
+            	if ( isStartOfXmlEntity(value,i) ) {
```

Comment:
Please delete blanks after `(` and before `)`, thanks.
```suggestion
            	if (isStartOfXmlEntity(value,i)) {
```